HX711.h

```diff
@@ -50,11 +50,11 @@ class HX711
   //  max timeout
   bool     wait_ready_timeout(uint32_t timeout = 1000, uint32_t ms = 0);
 
-  //  raw read
-  float    read();
-  //  get average of multiple raw reads
-  //  times = 1 or more
-  float    read_average(uint8_t times = 10);
+
+  float    read();//  raw read
+
+
+  float    read_average(uint8_t times = 10);//  get average of multiple raw reads //  times = 1 or more
```

Comment:
What is the purpose of moving comments?
It does not improve the functionality or the readability (in my opinion).

Reply:
I move them for my convenience, they don't need to stay

Reply:
Every person has its own preferred style of indenting, comments etc. Changing these, especially if done partially, disrupt the structure the maintainer of the code uses. This makes it harder for the maintainer to go through the code. As I try to keep all my 150++ libraries somehow consistent in terms of style - which is hard enough - I will not merge any PR that makes changes in comments or layout or style without a very good reason.

So please revert these changes.